index.html

```diff
@@ -349,6 +343,34 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>While the pointer is locked, and
+	   the element set as the target for mouse events is a
+           <a href="https://dom.spec.whatwg.org/#concept-shadow-including-descendant">shadow-including descendant</a> of the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+	   returns the result of the
+           <a href="https://https://w3c.github.io/webcomponents/spec/shadow/#dfn-retargeting-algorithm">retargeting algorithm</a>
+           with the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a> and
+	   the element as input, if the result and the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a> are in the
+           same tree, otherwise returns null.</p>
```

Comment:
Oops, as "target ... is a shadow-including descendant" already exists, so the example above is invalid.  Let me revise the case.